src/util.cpp

```diff
@@ -1019,16 +1019,16 @@ bool FileCommit(FILE *file)
         LogPrintf("%s: FlushFileBuffers failed: %d\n", __func__, GetLastError());
         return false;
     }
+#if defined(MAC_OSX) && defined(F_FULLFSYNC)
```

Comment:
It's impossible to both define WIN32 and MAC_OSX at the same time.

Reply:
This was meant to be an `#elif`, fixed.